brick/dockerlib.py

```diff
@@ -12,6 +13,36 @@
 docker_client = docker.from_env()
 
 
+def is_built_up_to_date(tag: str, dependency_paths: List[str]) -> bool:
+    """
+    Verifies if the given image tag is newer than all relative paths.
+    """
+    if dependency_paths is None:
+        return False
+
+    inspect_result = subprocess.run(
+        f"docker inspect -f '{{{{ json .Metadata.LastTagTime }}}}' {tag}",
+        shell=True,
+        capture_output=True)
```

Comment:
The CI server is running Python 3.6, so `capture_output` is not supported (it was introduced in 3.7) - could we try with a workaround to avoid updating the server while testing? :)
([here's a few suggestions](https://stackoverflow.com/questions/53209127/subprocess-unexpected-keyword-argument-capture-output/53209196))

Reply:
Should be fixed now. :)